src/main/java/org/broadinstitute/hellbender/utils/pairhmm/PairHMM.java

```diff
@@ -65,13 +65,28 @@
             logger.info("Using the OpenMP multi-threaded AVX-accelerated native PairHMM implementation");
             return hmm;
         }),
+        /* FPGA implementation of LOGLESS_CACHING called through JNI. Throws if FPGA is not available */
+        EXPERIMENTAL_FPGA_LOGLESS_CACHING(args -> {
+            // Constructor will throw a UserException if OpenMP AVX is not available
```

Comment:
Comments says throws if "OpenMP AVX" is not available, but comment above says "Throws if FPGA is not available" -- I assume the latter is correct?